iOS/ACPWeatherExample/ACPWeatherExample/WeatherExtensionInternal.m

```diff
@@ -193,12 +193,12 @@ - (void) processEvents {
         
         // NOTE: configuration is mandatory processing the event, so if shared state is null stop processing events
         if (!configSharedState) {
-            NSLog(@"%@ - Could not process event, configuration shared state is pending", [self name]);
+            [ACPCore log:ACPMobileLogLevelWarning tag:@"WeatherExtensionInternal" message:[NSString stringWithFormat:@"%@ - Could not process event, configuration shared state is pending", [self name]]];
```

Comment:
Same comment here, we can use log:ACPMobileLogLevelDebug for pending shared states.